src/chrome/content/rules/Cryptome.xml

```diff
@@ -1,20 +1,18 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://www.cryptome.org/ => https://www.cryptome.org/: (60, 'SSL certificate problem: unable to get local issuer certificate')
-
+	SSL error:
+		cryptome.info
+		www.cryptome.info
 -->
-<ruleset name="Cryptome.org" default_off="failed ruleset test">
-
-  <target host="cryptome.org" />
-  <target host="www.cryptome.org" />
+<ruleset name="Cryptome.org" >
+	<target host="cryptome.org" />
+	<target host="www.cryptome.org" />
 
-  <target host="cryptome.info" />
-  <target host="www.cryptome.info" />
+	<target host="cryptome.info" />
+	<target host="www.cryptome.info" />
 
-  <rule from="^http://(www\.)?cryptome\.info/"
-          to="https://cryptome.org/"/>
+	<securecookie host=".+" name=".+" />
 
-  <rule from="^http:" to="https:" />
+	<rule from="^http://(www\.)?cryptome\.info/" to="https://cryptome.org/"/>
```

Comment:
They do not have the same content

Reply:
@J0WI yes, but it says that urls with .info should be changed to .org instead of doing the automatic redirect, I see no problem wih this approach, no?

Reply:
@J0WI *ping*

Reply:
Please have a look at our policy: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects

Reply:
@J0WI thanks, pushed a fix

Reply:
Thanks!